ecs/file_format_build.tf

```diff
@@ -0,0 +1,98 @@
+resource "aws_ecs_task_definition" "file_format_build_task_definition" {
+  count                    = local.count_file_format_build
+  container_definitions    = templatefile("${path.module}/templates/file_format_build.json.tpl", { log_group_name = aws_cloudwatch_log_group.file_format_build_log_group[count.index].name, app_environment = local.environment })
+  family                   = "file-format-build-${local.environment}"
+  task_role_arn            = aws_iam_role.fileformat_ecs_task[count.index].arn
+  execution_role_arn       = aws_iam_role.fileformat_ecs_execution[count.index].arn
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = 512
+  memory                   = 1024
+  network_mode             = "awsvpc"
+  volume {
+    name = "fileformatbuild"
+    efs_volume_configuration {
+      file_system_id = var.file_system.id
+      root_directory = "/"
+      authorization_config {
+        iam             = "ENABLED"
+        access_point_id = var.access_point.id
+      }
+      transit_encryption = "ENABLED"
+    }
+  }
+}
+
+resource "aws_ecs_cluster" "file_format_build_cluster" {
+  count = local.count_file_format_build
+  name  = "file_format_build_${local.environment}"
+}
+
+resource "aws_iam_role" "fileformat_ecs_execution" {
+  count              = local.count_file_format_build
+  name               = "FileFormatECSExecutionRole${title(local.environment)}"
```

Comment:
Role is missing the "TDR" prefix. Maybe should be added for consistency?

Reply:
Yep, I've added it in here and for the policy. I've added in a variable for the project as well. I've set up these ECS modules to be the same as the lambdas but it's still worth having a project variable in based on your comment about the lambdas below.